Microsoft.Toolkit.Uwp.UI.Controls/Carousel/Carousel.xaml

```diff
@@ -10,6 +10,11 @@
                 </ItemsPanelTemplate>
             </Setter.Value>
         </Setter>
+        <Setter Property="EasingFunction">
```

Comment:
How will this affect users who have styled the control already?

Reply:
If they have not specified an easing function explicitly, the easing function will default to the default easing function for storyboard animations (I believe it's linear)